src/main/java/net/rpgtoolkit/common/assets/Item.java

```diff
@@ -242,6 +243,15 @@ public void setUsersSpecified(boolean usersSpecified) {
   }
 
   /**
+   * 
+   * @param userChar 
+   */
+  public void setUserChar(List<String> userChar) {

```

Comment:
I think we should probably consistently pluralize our collection names and the functions that go with them. setUserChars, equipLocations, etc.